### PR TITLE
Setup E2E testing for Apply

### PR DESCRIPTION
**Pull request by bryangaledxw**
# Changes in this PR

This PR adds E2E testing for submitting an application

# Release checklist

[Release process documentation](https://dsdmoj.atlassian.net/wiki/spaces/AP/pages/edit-v2/4247847062?draftShareId=a1c360ab-bd31-4db1-aae3-7cc002761de9).

## Pre-merge checklist

- [ ] Are any changes required to dependent services for this change to work?
  (eg. CAS API)
- [ ] Have they been released to production already?

## Post-merge checklist

- [ ] [Manually approve](https://dsdmoj.atlassian.net/wiki/spaces/AP/pages/4247847062/Release+process#Manual-releases)
  release to test
- [ ] [Manually approve](https://dsdmoj.atlassian.net/wiki/spaces/AP/pages/4247847062/Release+process#Manual-releases)
  release to preprod
- [ ] [Manually approve](https://dsdmoj.atlassian.net/wiki/spaces/AP/pages/4247847062/Release+process#Manual-releases)
  release to prod

### CODE REVIEW

**Comment by bryangaledxw** [comment]
[Setup E2E testing for Apply](https://trello.com/c/MJ9f4Nbt/816-setup-e2e-testing-for-apply)

**Comment by bryangaledxw** [comment]
[Setup E2E testing for Apply](https://trello.com/c/MJ9f4Nbt/816-setup-e2e-testing-for-apply)

File: cypress_shared/fixtures/offences-dev.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+[
+  {
```
Comment (tahb) [inline cypress_shared/fixtures/offences-dev.json:2]: Do we need to test with a certain CRN for these to work or are they more general than that?

**Review by tahb — APPROVED** [review]
👍

File: cypress_shared/fixtures/offences-dev.json (hunk 1 above)
Comment (bryangaledxw) [reply cypress_shared/fixtures/offences-dev.json:2]: Certain CRN. `offences-dev.json` and `offences-local.json` go with the person from `person-dev.json` and `person-local.json`

**Pull request merged**